test/index.test.js

```diff
@@ -3,7 +3,30 @@
 const assert = require('chai').assert;
 
 describe('index test', () => {
-    it('fails', () => {
-        assert.isTrue(false);
+    let instance;
+    let CoverageBase;
+
+    beforeEach(() => {
+        // eslint-disable-next-line global-require
+        CoverageBase = require('../index');
+        instance = new CoverageBase({});
+    });
+
+    afterEach(() => {
+        instance = null;
+    });
+
+    it('can create an coverage base class instance', () => {
+        assert.instanceOf(instance, CoverageBase);
+    });
+
+    it('should catch an error', () => {
```

Comment:
would be nice to nest under `describe('getUploadCoverageCmd',` and rename test to `returns not implemented`